app/models/manageiq/providers/ibm_power_vc/network_manager/floating_ip.rb

```diff
@@ -0,0 +1,4 @@
+ManageIQ::Providers::Openstack::NetworkManager::FloatingIP.include(ActsAsStiLeafClass)
```

Comment:
This is the wrong class name, fixed in https://github.com/ManageIQ/manageiq-providers-ibm_power_vc/pull/55